modular_chomp/code/modules/mob/living/simple_mob/subtypes/occult/constructs/juggernaut.dm

```diff
@@ -3,13 +3,16 @@
 	real_name = "Behemoth"
 	desc = "The pinnacle of occult technology, Behemoths are nothing shy of both an Immovable Object, and Unstoppable Force."
 	melee_damage_lower = 45
-	melee_damage_upper = 55
+	melee_damage_upper = 65
 	ai_holder_type = /datum/ai_holder/simple_mob/intentional/adv_dark_gygax
-	projectiletype = /obj/item/projectile/icicle
+	/obj/item/projectile/energy/electrode/cult

```

Comment:
projectiletype = /obj/item/projectile/energy/electrode/cult

Reply:
Ahh thats what I forgot. Opps, should be fixed now.